### ISSUE TITLE
Releasing v7

### ISSUE THREAD

**Issue by WikiRik** [sequelize/meetings#23]
I will update this issue later with more, but there's several things we could do upon releasing v7. Mainly we need to promote the new features and the new scoped package name.

We can do that through;
- Submitting us for the monthly GitHub Release Radar; https://github.com/github/release-radar
- Contacting [StackShare](https://stackshare.io/stackups/sequelize-vs-typeorm) and other websites to update their comparison between us and other JS ORMs

What we should also do;
- Contact the maintainers for the projects mentioned in the docs (Resources / Awesome Sequelize) to make sure their projects can work with v7 around the release date

Feel free to comment with more

**Comment by fzn0x** [comment]
@WikiRik I think creating a blog is good to promote new v7 features, so people can read and start using Sequelize with no hassle.

**Comment by fzn0x** [comment]
Came across and I think it's related to https://github.com/sequelize/website/issues/125

**Comment by ephys** [comment]
I agree blog posts would be great. We should showcase them in our upgrade guide to motivate people to migrate

**Comment by fzn0x** [comment]
> I agree blog posts would be great. We should showcase them in our upgrade guide to motivate people to migrate
> 
> Edit: loving your new profile picture

Thank you!

**Comment by ephys** [comment]
Done, all blocked things are postponed to v8

**Comment by WikiRik** [comment]
The first beta of v7 is getting close which means we should start thinking about how to involve users.

First step would probably be a pinned issue explaining what it means that a beta is out and what our plans are until we release a full version. In that pinned issue we'll highlight the biggest changes we've done (TypeScript, DataTypes, package rename) and that we would like user feedback on it. We'll mention that we want to avoid breaking changes in the betas so users can start to migrate projects already without the fear of changing too much when the next beta comes out. Finally we also mention that apart from reporting bugs we also welcome specific feedback on the upgrade guides of both v6 and v7 so more people will migrate to v7.

After that we should contact maintainers from the projects mentioned here; https://sequelize.org/docs/v7/other-topics/resources/ (and possibly update that list) telling them that v7 is coming up and we'd like it if they update their projects to support v7.

Once that is in full motion, we can start on further outreach like the aforementioned blog posts and making other plans for the real release of v7.0.0.
